plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -1935,44 +2060,69 @@ void producer_plugin_impl::log_trx_results( const packed_transaction_ptr& trx,
    bool except = except_ptr || (trace && trace->except);
    if (except) {
       if (_pending_block_mode == pending_block_mode::producing) {
-         fc_dlog(_trx_log, "[TRX_TRACE] Block ${block_num} for producer ${prod} is REJECTING tx: ${trx}",
+         if ( is_transient ) {
+            fc_dlog(_transient_trx_failed_trace_log, "[TRX_TRACE] Block ${block_num} for producer ${prod} is REJECTING transient tx: ${txid}, auth: ${a}, ${details}",
+                 ("block_num", chain.head_block_num() + 1)("prod", get_pending_block_producer())("txid", trx->id())
+                 ("a", trx->get_transaction().first_authorizer())
+                 ("details", get_detailed_contract_except_info(trx, trace, except_ptr)));
```

Comment:
This could be combined with the other as:
```
            fc_dlog( is_transient ? _transient_trx_failed_trace_log : _trx_failed_trace_log, 
                     "[TRX_TRACE] Block ${block_num} for producer ${prod} is REJECTING ${desc}tx: ${txid}, auth: ${a}, ${details}",
                 ("block_num", chain.head_block_num() + 1)("prod", get_pending_block_producer())
                 ("desc", is_transient ? "transient " : "")("txid", trx->id())
                 ("a", trx->get_transaction().first_authorizer())
                 ("details", get_detailed_contract_except_info(trx, trace, except_ptr)));
```
Similar for others.

Reply:
Thanks. This removes duplicate code.